src/main/resources/portal.properties.EXAMPLE

```diff
@@ -191,6 +191,8 @@ oncokb.public_api.url=https://public.api.oncokb.org/api/v1
 # Your OncoKB Token which an be found under https://www.oncokb.org/account/settings
 oncokb.token=
 
+# Enable merging of OncoKB icons
+oncokb.merge_icons=true
```

Comment:
i think it's better to comment out this one. The default of `true` should be done in code rather than in the properties file. Issue is that when someone upgrades their cbioportal version they wouldn't have this property set in their existing properties file